src/Sulu/Component/Webspace/NavigationContext.php

```diff
@@ -43,11 +43,11 @@ public function getKey()
     }
 
     /**
-     * @param $locale
+     * @param string $locale
      *
      * @return null|string
      */
-    public function getTitle($locale)
+    public function getTitle(string $locale)
```

Comment:
This is also duplicated information... I would say remove the type in this line, to be absolutely sure that we don't make a BC break now.